### ISSUE TITLE
spython fail on load shell

### ISSUE THREAD

**Issue by al3x609** [singularityhub/singularity-cli#55]
hi, for test I run:

`spython shell`

output:
```
Traceback (most recent call last):
  File "/usr/local/bin/spython", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/spython/client/__init__.py", line 167, in main
    main(args=args, options=options, parser=parser)
  File "/usr/local/lib/python3.7/site-packages/spython/client/shell.py", line 40, in main
    return lookup[shell](image)
  File "/usr/local/lib/python3.7/site-packages/spython/client/shell.py", line 90, in python
    code.interact(local={"client":cli})
NameError: name 'cli' is not defined ```

Can you help me with that?

**Comment by vsoch** [comment]
Yeah!  This is a bug and an easy fix. I’m out on a run but I should be back in a couple of hours to make the fix! In the meantime you can actually just install ipython and it should find that environment and use it instead (I just looked at the code and I don’t see the same bug there).

**Comment by vsoch** [comment]
hey @al3x609 I have a fix for you here! https://github.com/singularityhub/singularity-cli/pull/56

Would you care to test? You should be able to clone the branch, install and then do a shell (without ipython)

```
git clone -b fix cli-bug https://github.com/singularityhub/singularity-cli
cd singularity-cli
python setup.py install
```
When tests pass (and you approve) this will be released as 0.0.40 on pypi.

**Comment by al3x609** [comment]
Hi, it  works  ;) 
thanks for your prompt response
